os/kernel/binary_manager/Make.defs

```diff
@@ -24,7 +24,7 @@ CSRCS += binary_manager.c binary_manager_getinfo.c binary_manager_response.c
 CSRCS += binary_manager_data.c binary_manager_bootparam.c
 
 ifeq ($(CONFIG_APP_BINARY_SEPARATION),y)
-CSRCS += binary_manager_load.c binary_manager_callback.c
+CSRCS += binary_manager_load.c binary_manager_callback.c binary_manager_deinit.c
```

Comment:
Is `binary_manager_deinit.c` needed even if there is no recovery?
I think deinit is needed only when binary manager tries to reload the modules.

Reply:
It is needed for binary update also.